help/examples/deconvolution_example.pro

```diff
@@ -0,0 +1,91 @@
+;;;
+;;; Examples of using both FOXSI deconvolution methods.
+;;;
+
+;
+; The "simple" method (just a wrapper for max_likelihood.pro)
+;
+
+foxsi, 2014
+file = 'foxsi2_flare1_det6_t1pos0.sav'
```

Comment:
@jtvievering  This example uses a flare image that you made.  I have it in a save file.  It would be useful if you can paste in the few lines of code that you used to make that image.  Doesn't need to be position 0, but if you change it then make sure to update which PSF is restored.